### ISSUE TITLE
Newline after opening parenthesis/bracket of if statement where there's multiple conditions flagged as error

### ISSUE THREAD

**Issue by jian-wu** [squizlabs/PHP_CodeSniffer#459]
Hi, version 2.2 reports newline after open parenthesis/bracket as error when version 2.1 didn't.  Please see example below.

```
if (
    $longConditionA == $longConditionB
    && $longConditionC == $longConditionD
    && $longConditionE == $longConditionF
) {
    // do something
} else {
    // do something else
}
```
The above is changed to this below after applying `phpcbf --standard=PSR2`
```
if ($longConditionA == $longConditionB
    && $longConditionC == $longConditionD
    && $longConditionE == $longConditionF
) {
    // do something
} else {
    // do something else
}
```
But according to PSR-2, section 4.6:
>Argument lists MAY be split across multiple lines, where each subsequent line is indented once. When doing so, the first item in the list MUST be on the next line, and there MUST be only one argument per line.

Although there's no specific rule for control structures, I'm applying the same style guide just for better readability and consistency.  IMHO, this should not have been flagged as an error.

Thanks.

**Comment by aik099** [comment]
Clearly it's not argument list here.

**Comment by jian-wu** [comment]
Hi @aik099, 

Which one do you think is more readable and more consistent compared to other PSR-2 standards?

Thanks.

**Comment by aik099** [comment]
Because of PSR-2 has some many things not covered by standard I tend to use my own standard.

I think fixed version looks better. But since there is no rule of thumb here or in the standard itself then I guess both options are correct and fixed version is preferred by PHP_CodeSniffer before it not only fixes it, but also reports original code as error.

**Comment by jian-wu** [comment]
Can it be a warning rather than an error since it's not an error?  Thanks.

**Comment by aik099** [comment]
It will be auto-fixed regardless of it's warning/error status. Let's wait @gsherwood opinion on this.

@jian-wu , if you want that particular error not being reported/fixed then you can create custom standard based on PSR-2 and exclude that particular sniff/error.

**Comment by jian-wu** [comment]
OK.  I can look into that.  The reason why I ask to have it as a warning is because I ignore warnings and choose to not fix it until further determination.  For instance, the long line warnings are usually not fixable.  Let's see what @gsherwood has to say on this.  Thanks again.

**Comment by jian-wu** [comment]
I'm closing this issue since it's just an opinionated one.  Thanks.

**Comment by gsherwood** [comment]
This is a multi-line IF statement, which PSR-2 does not consider. It considers IF statements to all have the same rules: https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-2-coding-style-guide.md#5-control-structures

Also see the example IF code a little bit further down.

The specific rule PHPCS is enforcing here is: `There MUST NOT be a space after the opening parenthesis`

I realised that older versions were not enforcing and, more importantly, not fixing this rule. This was corrected in 2.2.
